go.mod

```diff
@@ -4,38 +4,36 @@ go 1.11
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/Microsoft/go-winio v0.4.14
+	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
+	github.com/Microsoft/hcsshim v0.8.7 // indirect
 	github.com/apache/thrift v0.0.0-20161221203622-b2a4d4ae21c7 // indirect
 	github.com/codahale/hdrhistogram v0.0.0-20160425231609-f8ad88b59a58 // indirect
-	github.com/containerd/cgroups v0.0.0-20190717030353-c4b9ac5c7601 // indirect
 	github.com/containerd/console v0.0.0-20181022165439-0650fd9eeb50
-	github.com/containerd/containerd v1.4.0-0.20191014053712-acdcf13d5eaf
+	github.com/containerd/containerd v1.3.1-0.20191217142032-9b5581cc9c5b
```

Comment:
This will break for example when fsutil https://github.com/tonistiigi/fsutil/blob/master/go.mod#L6 is updated to `1.3.2` . Same for other similar projects where we used pseudo versions. Unfortunately, I don't know any way to make this work without adding `rewrite` rules to all of such dependencies (that was recommended to us by the people who broke go1.13 when we explained this issue). And if we add rewrites then people still can't depend on our mod file directly. At least atm they get an error when they forget to add a rewrite rule.